src/content/docs/browser/new-relic-browser/browser-pro-features/filterable-geography-webpage-metrics-location.mdx

```diff
@@ -62,7 +54,7 @@ To view or sort your browser monitoring performance data by location:
 <img
   title="Filterable geography charts UI"
   alt="Screenshot of Filterable geography charts UI in New Relic"
-  src={BrowserFilterGeos2}
+  src={browserGeoData}
```

Comment:
@a-sassman looks like you're missing an import statement for this one.

Reply:
I fixed this with the latest commit.